static/js/admin.js

```diff
@@ -141,7 +141,41 @@ exports.setup_page = function () {
         $("#deactivation_stream_modal").modal("show");
     });
 
-    $(".admin_user_table").on("click", ".reactivate", function (e) {
+    $(".admin_bot_table").on("click", ".deactivate", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+
+        $(".active_user_row").removeClass("active_user_row");
+
+        $(e.target).closest(".user_row").addClass("active_user_row");
```

Comment:
We select this jquery object a few more times including the success callback where we call it `row`. We should be able to select it once.

It seems we reuse the user template to render bots so the CSS selectors have `user` in them which is unfortunate but I don't think cleaning it up should block this PR.